hoomd/data/array.py

```diff
@@ -730,11 +730,11 @@ class HOOMDGPUArray(_NoGPU):
 The HOOMDGPUArray object exposes a GPU data buffer using the
 `__cuda_array_interface__
 <https://numba.pydata.org/numba-doc/latest/cuda/cuda_array_interface.html>`_.
-This class tries to prevent invalid memory access through only allow the buffer
-to be exposed within a context manager (`hoomd.State.gpu_local_snapshot`).
-However, to prevent copying the data, we cannot guarentee the data will not
-leak. This means that it is possible to *escape* this class. An example of an
-error of this kind is shown below.
+This class tries to prevent invalid memory access through only allowing the
+buffer to be exposed within a context manager
+(`hoomd.State.gpu_local_snapshot`). However, to prevent copying the data, we
+cannot guarentee the data will not leak. This means that it is possible to
+*escape* this class. An example of an error of this kind is shown below.
```

Comment:
```suggestion
This class provides buffer access through a context manager to prevent invalid
memory accesses (`hoomd.State.gpu_local_snapshot`). To avoid errors, do not use
references to the data outside the context manager. For example:
```
Fewer words for clarity.